packages/xchain-radix/src/const.ts

```diff
@@ -0,0 +1,32 @@
+import { Network, RootDerivationPaths } from '@xchainjs/xchain-client/lib'
+import { Asset } from '@xchainjs/xchain-util/lib'
+
+/**
+ * Chain identifier for Radix.
+ * This constant represents the identifier for the Radix Chain.
+ */
+export const RadixChain = 'RADIX' as const
+
+export const MAINNET_GATEWAY_URL = 'https://mainnet.radixdlt.com'
+export const STOKENET_GATEWAY_URL = 'https://stokenet.radixdlt.com'
+
+export const XRD_DECIMAL = 18
+
+export const XrdAsset: Asset = {
+  symbol: 'resource_rdx1tknxxxxxxxxxradxrdxxxxxxxxx009923554798xxxxxxxxxradxrd',
+  ticker: 'XRD',
+  synth: false,
+  chain: RadixChain,
+}
+
+export const xrdRootDerivationPaths: RootDerivationPaths = {
```

Comment:
Few things:

* Is the `2'` here true? I think that this is the network Id and that it should be `1'`.
* Why is the `TestNet` using a network id of `12`? The testnet is stokenet and should be `2`.

Reply:
first issue is fixed.
second issue, I mapped Stagenet to Stokenet and Testnet to Kisharnet. I need Kisharnet for the uni tests that verify that derived addresses match the wallet addresses

Reply:
I think that we should still define the testnet to be stokenet to be consistent with all other parts of the code